cmd/wasm-client/go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/lightninglabs/faraday v0.2.7-alpha.0.20220503144421-cd1e56982f09
 	github.com/lightninglabs/lightning-node-connect v0.1.9-alpha.0.20220602120524-e9964c685b18
-	github.com/lightninglabs/loop v0.18.0-beta
+	github.com/lightninglabs/loop v0.18.0-beta.0.20220602162617-992538eed917
```

Comment:
Should we also do a "pretty tag" or is the next tag for loop not quite scheduled yet?